ibl_to_nwb/metadata/ecephys.yml

```
@@ -82,3 +82,7 @@ Ecephys:
       description: Proportion of quality metrics passed.
     - name: allen_location
       description: Brain region reference in the Allen Mouse Brain Atlas.
+    - name: uuid
+      description: Unique identifier given to each cluster after spike sorting. Enables comparison of each cluster in this .nwb file to its counterpart in ONE (see https://int-brain-lab.github.io/ONE/index.html)
```

Putting URLs in metadata descriptions can be very brittle (imagine, these files will live on DANDI once published until 'the end of time') since the website can easily change as a live service - so I generally discourage this

```suggestion
      description: Unique identifier given to each cluster after spike sorting. Enables comparison of each cluster in this .nwb file to its counterpart in ONE.
```